### ISSUE TITLE
FacetingSettings: Deserialize<'de>` is not satisfied

### ISSUE THREAD

**Issue by JaydenElliott** [meilisearch/meilisearch-rust#446]
**Description**
I have spun up an empty project with meilisearch-sdk 0.22, no other libs. When trying to build I get the following error
```
error: unexpected end of input, unexpected token in nested attribute, expected ident
  --> /home/jaydenelliott/.cargo/registry/src/github.com-1ecc6299db9ec823/meilisearch-sdk-0.22.0/src/settings.rs:19:13
   |
19 |     #[serde()]
   |             ^

error[E0277]: the trait bound `for<'de> FacetingSettings: Deserialize<'de>` is not satisfied
   --> /home/jaydenelliott/.cargo/registry/src/github.com-1ecc6299db9ec823/meilisearch-sdk-0.22.0/src/settings.rs:538:27
    |
538 |         request::<(), (), FacetingSettings>(
    |                           ^^^^^^^^^^^^^^^^ the trait `for<'de> Deserialize<'de>` is not implemented for `FacetingSettings`
```

**Environment (please complete the following information):**
- PopOs
- meilisearch-rust version: 0.22

**Comment by bidoubiwa** [comment]
Hey @JaydenElliott

Indeed, this is due to a new serde patch version that breaks our build.
It is fixed here #448.

**Comment by bidoubiwa** [comment]
Released failed 😅
